src/content/en/tools/lighthouse/audits/has-viewport-meta-tag.md

```diff
@@ -1,11 +1,11 @@
 project_path: /web/_project.yaml
 book_path: /web/tools/_book.yaml
-description: Reference documentation for the "HTML Has a Viewport Meta Tag" Lighthouse audit.
+description: Reference documentation for the "Has A <meta name="viewport"> Tag With width Or initial-scale" Lighthouse audit.
```

Comment:
DevSite misbehaves when there are HTML tags in the description, I'd suggest either encoding, or changing to something like "Has a meta viewport Tag..."